docs/man/git-lfs-config.5.ronn

```diff
@@ -405,6 +405,7 @@ including and limited to:
 - lfs.url
 - remote.{name}.lfsurl
 - lfs.{*}.access
+- lfs.skipdownloaderrors
```

Comment:
In this case, we might also fix the ordering of `lfs.{*}.access` and `remote.{*}.access` while we're at it.

```suggestion
- lfs.skipdownloaderrors
- lfs.url
- lfs.{*}.access
- remote.{name}.lfsurl
```